rasa/utils/io.py

```diff
@@ -121,7 +134,28 @@ def read_yaml(content: Text) -> Union[List[Any], Dict[Text, Any]]:
             .decode("utf-16")
         )
 
-    return yaml_parser.load(content) or {}
+    return content
+
+
+def change_sections_in_yaml_file(data: Dict[Text, Any], filename: Text) -> None:
+    """Changes specific sections (given by keys) in a yaml file.
+
+     Args:
+        data: The keys are the sections to be changed, the values the contents of the
+              respective sections.
+        filename: Name of the file to be changed.
+    """
+    old_content = read_file(filename, DEFAULT_ENCODING)
```

Comment:
there is the slight chance that the file got deleted in between or we can't access it for some other reasons.